src/Umbraco.Core/Services/MemberService.cs

```diff
@@ -399,6 +399,19 @@ public IEnumerable<IMember> GetAll(long pageIndex, int pageSize, out long totalR
             return _memberRepository.Get(query)?.FirstOrDefault();
         }
 
+        /// <summary>
+        /// Get an list of <see cref="IMember"/> for all members with the specified email.
+        /// </summary>
+        /// <param name="email">Email to use for retrieval</param>
+        /// <returns><see cref="IEnumerable{IMember}"/></returns>
+        public IEnumerable<IMember> GetMembersByEmail(string email)
```

Comment:
The `GetByEmail()` method can call this method using `=> GetMembersByEmail(email).FirstOrDefault()`  to remove duplicate code.